generators/app/files.js

```diff
@@ -176,6 +176,7 @@ function writeFiles(generator) {
 
         generator.entityClass = entity;
         generator.camelCaseEntityClass = _.camelCase(entity);
+        generator.metaType = useEntityAsType ? 'Entity' : 'String';
```

Comment:
Linked with comments in `StringDeserializer.java.ejs` and `StringSerializer.java.ejs`

Reply:
You should use only `type` in existing (de)serializers templates IMO